### DISCUSSION TITLE
[Panduan] Index Forum List🧾

### DISCUSSION THREAD

**Discussion by deaafrizal** [deaafrizal/forum#1] [Announcements]
# Deadev Forum, Lorem Ipsum! 👋

_**disclaimer: please, disini gak harus selalu "bang dea, tanya".
bisa dibuat secara global aja ya kalau bertanya, forum ini judulnya diskusi, sharing dan tidak selalu terkait nama gue.**_

## Index Forum (kumpulan kategori)
  * tanya problem teknis & non teknis di [troubleshooting room](https://github.com/deaaprizal/forum/discussions/categories/troubleshooting-room)
  * chill nongki santai di [the lounge](https://github.com/deaaprizal/forum/discussions/categories/the-lounge)
  * share projekan yang lagi dikerjain buat jadi referensi dan penyemangat developer lain di [my projects](https://github.com/deaaprizal/forum/discussions/categories/my-projects)
  * sharing seputar peralatan tempur developer [all about hardware for developer](https://github.com/deaaprizal/forum/discussions/categories/all-about-hardware-pc-laptop-etc)

homebase: https://github.com/deaaprizal/forum

## Authors

- [@deaafrizal](https://instagram.com/dea.afrizal)
- [...?]()

### Moderator

- [...?]()
- [...?]()
- [...?]()

**Comment by IrnandaNanda** [comment]
Halo

**Comment by regezzmcn** [comment]
Bang coba buat grup wa atau grup telegram aja bang biar simpel buat saling berbagi atau bertukar pendapat 🙏🙏

**Reply by bagasstrongman** [reply]
Setuju

**Reply by JsStudio93** [reply]
Iya bang

**Reply by esqurianto-fai** [reply]
setuju banget

**Reply by mwafa** [reply]
menurutku lebih bagus di github, kalau group chat gitu jadinya topiknya numpuk.
kalau ini bisa ngobrol per topik.

dan bisa dibaca yang lagi cari solusi juga.

**Comment by ghost** [comment]
Hallo cuyy

**Comment by chenjacky32** [comment]
Test

**Comment by YusufJago** [comment]
Test

**Comment by uratmangun** [comment]
@deaaprizal gak ada grup telegram bang?

**Comment by AlBadar20** [comment]
Bang ada tutorial terkait cara membuat aplikasi report akun Instagram privat?🙏🏻

**Comment by adzka54321** [comment]
Mantap

**Comment by jinxproNash** [comment]
Bang boleh dong buat DC biar bisa diskusi mau dalemin linux karena kebutuhan kerja

**Comment by Mistyy266** [comment]
halo semuanya , gua baru mulai belajar coding, salken semuanya

**Reply by SaityNotFound** [reply]
Salken juga

**Comment by ryanazryan** [comment]
halo semua nya 👋
